playbooks/roles/genesis-google/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
These are getting set in the main playbook too. So we probably don't need them defined here as well.

This is reminding me (again) that I need to refactor how to turn L2TP/IPsec off for Amazon and GCE. A parameterized include with a conditional is a much better way to disable L2TP/IPsec. That will probably be one of the first things I do after this is merged.